src/components/Detail/_detail.scss

```diff
@@ -10,8 +11,21 @@
   @include text($string2);
 
   &__label {
+    flex-shrink: 0;
     font-variation-settings: 'wght' 600;
```

Comment:
this is not a blocking thing, but typically we go the other way mobile at the top, then override at the higher breakpoints using the `@include media($size-md) {`